app/controllers/api/v2/compliance/arf_reports_controller.rb

```diff
@@ -31,7 +31,7 @@ def get_resource
         end
 
         resource_description do
-          resource_id 'foreman_openscap_arf_reports'
+          resource_id 'arf_reports'
```

Comment:
I'm not apipie expert, but based on documentation, this should not be needed anymore? https://github.com/Apipie/apipie-rails#resource-description

@shlomizadok do you happen to know what was the reason for custom resource_id?